### PR TITLE
Digital Business Card button redirection

### PR DESCRIPTION
**Pull request by patrickpeinanw**
*Issue:*https://github.com/bcgov/entity/issues/21697

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-31-u93zf9n7.web.app

File: src/components/bcros/businessDetails/LinkActions.vue [hunk 1]
```diff
@@ -84,6 +89,11 @@ const actions: ComputedRef<Array<Array<MenuActionItem>>> = computed(() => {
   return [allowedActions]
 })
 
+const redirectToDigitalCredentialsPage = () => {
```
Comment (hfekete) [inline src/components/bcros/businessDetails/LinkActions.vue:92]: can you move this redirect to the redirect composable ? :D
As its composable, all the config and store things should work there too.

**Review by hfekete — APPROVED** [review]
Nice, thanks. 
Had one comment, if you can't do it, that's OK too.

**Pull request merged**